packages/router/src/router.tsx

```diff
@@ -68,34 +63,28 @@ function Route(props: RouteProps | RedirectRouteProps | NotFoundRouteProps) {
 
 const InternalRoute: React.VFC<InternalRouteProps> = ({
   path,
-  page,
   name,
   redirect,
   notfound,
 }) => {
   const routerState = useRouterState()
   const activePageContext = useActivePageContext()
 
+  /** The not-found route is handled by the router. */
   if (notfound) {
-    // The "notfound" route is handled by <NotFoundChecker>
     return null
   }
 
-  if (!path) {
-    throw new Error(`Route "${name}" needs to specify a path`)
-  }
-
-  // Check for issues with the path.
-  validatePath(path)
+  path as string
```

Comment:
I added this and other assertions to illustrate what I expect. At this point, path should be defined, since if a route isn't the not-found route, it has to either be a route or a redirect route (path is required for both).

Reply:
Having a type assertion on a line of its own like that, does it actually do anything?